使用PostgreSQL数据库，第二次启动失败

使用 docker compose 启动。

配置文件：
docker-compose.yml
```
version: '3.5'

services:
  nacos:
    image: nacos/nacos-server:v3.2.0
    volumes:
      - ./application-postgres.properties:/home/nacos/conf/application.properties
      - ./logs/:/home/nacos/logs/
    env_file:
      - docker-env-postgres.txt
    ports:
      - "32080:8080"
      - "8848:8848"
      - "9848:9848"
      - "9849:9849"
```

docker-env-postgres.txt
```
PREFER_HOST_MODE=hostname
MODE=standalone
SPRING_DATASOURCE_PLATFORM=postgresql
MYSQL_SERVICE_HOST=192.168.0.1
MYSQL_SERVICE_DB_NAME=D1
MYSQL_SERVICE_PORT=5432
MYSQL_SERVICE_USER=postgres
MYSQL_SERVICE_PASSWORD=postgres
MYSQL_SERVICE_DB_PARAM=currentSchema=nacos&autoReconnect=true&serverTimezone=PRC&characterEncoding=utf8&useSSL=true&tinyInt1isBit=false&useUnicode=true&zeroDateTimeBehavior=convertToNull&nullCatalogMeansCurrent=true&rewriteBatchedStatements=true&stringtype=unspecified
DB_DRIVER_CLASS_NAME=org.postgresql.Driver
NACOS_AUTH_IDENTITY_KEY=MTIzNDU2Nzg5MDEyMzQ1Nzg5MDEyMzQ1Njc4OTAxMjM0NTc4OTAxMjM0NTY3ODkwMTIzNDU3ODkw
NACOS_AUTH_IDENTITY_VALUE=MTIzNDU2Nzg5MDEyMzQ1Nzg5MDEyMzQ1Njc4OTAxMjM0NTc4OTAxMjM0NTY3ODkwMTIzNDU3ODkw
NACOS_AUTH_TOKEN=MTIzNDU2Nzg5MDEyMzQ1Nzg5MDEyMzQ1Njc4OTAxMjM0NTc4OTAxMjM0NTY3ODkwMTIzNDU3ODkw
TZ=Asia/Shanghai
```

从容器内把 application.properties 复制出来进行修改
application-postgres.properties
```
# ....
# 仅仅只增加这一行配置
db.pool.config.driverClassName=${DB_DRIVER_CLASS_NAME:com.mysql.cj.jdbc.Driver}
# ....
```

第一次 docker compose up 启动正常，能够访问控制台， CTRL+C 停止后，再次 docker compose up 就会失败无法启动。如果把数据库的数据全部清空掉，再次 docker compose up 还是会正常。

日志 docker compose up 命令控制台日志：
```
nacos-1  | 2026-04-01 12:10:15,132 WARN Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'configChangeClusterSyncRequestHandler' defined in URL [jar:nested:/home/nacos/target/nacos-server.jar/!BOOT-INF/lib/nacos-config-3.2.0.jar!/com/alibaba/nacos/config/server/remote/ConfigChangeClusterSyncRequestHandler.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'configMigrateService': Invocation of init method failed
nacos-1  | 
nacos-1  | 2026-04-01 12:10:15,150 WARN [WatchFileCenter] start close
nacos-1  | 
nacos-1  | 2026-04-01 12:10:15,151 WARN [WatchFileCenter] start to shutdown this watcher which is watch : /home/nacos/conf
nacos-1  | 
nacos-1  | 2026-04-01 12:10:15,152 WARN [WatchFileCenter] already closed
nacos-1  | 
nacos-1  | 2026-04-01 12:10:15,153 INFO [NotifyCenter] Start destroying Publisher
nacos-1  | 
nacos-1  | 2026-04-01 12:10:15,153 INFO [NotifyCenter] Completed destruction of Publisher
nacos-1  | 
nacos-1  | 2026-04-01 12:10:15,154 ERROR Startup errors : 
nacos-1  | 
nacos-1  | org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'configChangeClusterSyncRequestHandler' defined in URL [jar:nested:/home/nacos/target/nacos-server.jar/!BOOT-INF/lib/nacos-config-3.2.0.jar!/com/alibaba/nacos/config/server/remote/ConfigChangeClusterSyncRequestHandler.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'configMigrateService': Invocation of init method failed
nacos-1  |      at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:804)
nacos-1  |      at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:240)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1395)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1232)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:569)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:529)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:339)
nacos-1  |      at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:373)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:337)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
nacos-1  |      at org.springframework.beans.factory.support.DefaultListableBeanFactory.instantiateSingleton(DefaultListableBeanFactory.java:1221)
nacos-1  |      at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingleton(DefaultListableBeanFactory.java:1187)
nacos-1  |      at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:1123)
nacos-1  |      at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:987)
nacos-1  |      at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:627)
nacos-1  |      at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:752)
nacos-1  |      at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:439)
nacos-1  |      at org.springframework.boot.SpringApplication.run(SpringApplication.java:318)
nacos-1  |      at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:149)
nacos-1  |      at com.alibaba.nacos.bootstrap.NacosBootstrap.startCoreContext(NacosBootstrap.java:96)
nacos-1  |      at com.alibaba.nacos.bootstrap.NacosBootstrap.startWithConsole(NacosBootstrap.java:84)
nacos-1  |      at com.alibaba.nacos.bootstrap.NacosBootstrap.main(NacosBootstrap.java:54)
nacos-1  |      at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
nacos-1  |      at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
nacos-1  |      at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
nacos-1  |      at java.base/java.lang.reflect.Method.invoke(Method.java:569)
nacos-1  |      at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:102)
nacos-1  |      at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:64)
nacos-1  |      at org.springframework.boot.loader.launch.PropertiesLauncher.main(PropertiesLauncher.java:580)
nacos-1  | Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'configMigrateService': Invocation of init method failed
nacos-1  |      at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:222)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsBeforeInitialization(AbstractAutowireCapableBeanFactory.java:429)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1818)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:607)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:529)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:339)
nacos-1  |      at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:373)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:337)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:315)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
nacos-1  |      at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:254)
nacos-1  |      at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1760)
nacos-1  |      at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1643)
nacos-1  |      at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:913)
nacos-1  |      at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:791)
nacos-1  |      ... 28 common frames omitted
nacos-1  | Caused by: java.lang.Exception: [migrate] config_info namespace migrate insert failed
nacos-1  |      at com.alibaba.nacos.config.server.service.ConfigMigrateService.doCheckNamespaceMigrate(ConfigMigrateService.java:533)
nacos-1  |      at com.alibaba.nacos.config.server.service.ConfigMigrateService.migrate(ConfigMigrateService.java:157)
nacos-1  |      at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
nacos-1  |      at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
nacos-1  |      at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
nacos-1  |      at java.base/java.lang.reflect.Method.invoke(Method.java:569)
nacos-1  |      at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMethod.invoke(InitDestroyAnnotationBeanPostProcessor.java:457)
nacos-1  |      at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeInitMethods(InitDestroyAnnotationBeanPostProcessor.java:401)
nacos-1  |      at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:219)
nacos-1  |      ... 43 common frames omitted
nacos-1  | 2026-04-01 12:10:15,154 ERROR Nacos failed to start, please see /home/nacos/logs/nacos.log for more details.
nacos-1  | 
nacos-1  | 2026-04-01 12:10:15,166 INFO 
nacos-1  | 
nacos-1  | Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
nacos-1  | 
nacos-1  | 2026-04-01 12:10:15,185 ERROR Application run failed
nacos-1  | 
nacos-1  | org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'configChangeClusterSyncRequestHandler' defined in URL [jar:nested:/home/nacos/target/nacos-server.jar/!BOOT-INF/lib/nacos-config-3.2.0.jar!/com/alibaba/nacos/config/server/remote/ConfigChangeClusterSyncRequestHandler.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'configMigrateService': Invocation of init method failed
nacos-1  |      at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:804)
nacos-1  |      at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:240)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1395)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1232)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:569)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:529)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:339)
nacos-1  |      at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:373)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:337)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
nacos-1  |      at org.springframework.beans.factory.support.DefaultListableBeanFactory.instantiateSingleton(DefaultListableBeanFactory.java:1221)
nacos-1  |      at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingleton(DefaultListableBeanFactory.java:1187)
nacos-1  |      at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:1123)
nacos-1  |      at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:987)
nacos-1  |      at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:627)
nacos-1  |      at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:752)
nacos-1  |      at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:439)
nacos-1  |      at org.springframework.boot.SpringApplication.run(SpringApplication.java:318)
nacos-1  |      at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:149)
nacos-1  |      at com.alibaba.nacos.bootstrap.NacosBootstrap.startCoreContext(NacosBootstrap.java:96)
nacos-1  |      at com.alibaba.nacos.bootstrap.NacosBootstrap.startWithConsole(NacosBootstrap.java:84)
nacos-1  |      at com.alibaba.nacos.bootstrap.NacosBootstrap.main(NacosBootstrap.java:54)
nacos-1  |      at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
nacos-1  |      at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
nacos-1  |      at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
nacos-1  |      at java.base/java.lang.reflect.Method.invoke(Method.java:569)
nacos-1  |      at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:102)
nacos-1  |      at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:64)
nacos-1  |      at org.springframework.boot.loader.launch.PropertiesLauncher.main(PropertiesLauncher.java:580)
nacos-1  | Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'configMigrateService': Invocation of init method failed
nacos-1  |      at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:222)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsBeforeInitialization(AbstractAutowireCapableBeanFactory.java:429)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1818)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:607)
nacos-1  |      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:529)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:339)
nacos-1  |      at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:373)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:337)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:315)
nacos-1  |      at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
nacos-1  |      at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:254)
nacos-1  |      at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1760)
nacos-1  |      at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1643)
nacos-1  |      at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:913)
nacos-1  |      at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:791)
nacos-1  |      ... 28 common frames omitted
nacos-1  | Caused by: java.lang.Exception: [migrate] config_info namespace migrate insert failed
nacos-1  |      at com.alibaba.nacos.config.server.service.ConfigMigrateService.doCheckNamespaceMigrate(ConfigMigrateService.java:533)
nacos-1  |      at com.alibaba.nacos.config.server.service.ConfigMigrateService.migrate(ConfigMigrateService.java:157)
nacos-1  |      at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
nacos-1  |      at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
nacos-1  |      at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
nacos-1  |      at java.base/java.lang.reflect.Method.invoke(Method.java:569)
nacos-1  |      at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMethod.invoke(InitDestroyAnnotationBeanPostProcessor.java:457)
nacos-1  |      at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeInitMethods(InitDestroyAnnotationBeanPostProcessor.java:401)
nacos-1  |      at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:219)
nacos-1  |      ... 43 common frames omitted
nacos-1  | 2026-04-01 12:10:15,188 INFO [ThreadPoolManager] Start destroying ThreadPool
```

logs/nacos.log 日志
```

2026-04-01 12:20:30,504 INFO Nacos Server is starting...

2026-04-01 12:20:30,580 INFO ConfigQueryHandlerChain has been initialized successfully with chain: nacos

2026-04-01 12:20:30,886 INFO Nacos-related cluster resource initialization

2026-04-01 12:20:30,893 INFO Load com.alibaba.nacos.naming.ability.NamingAbilityInitializer for ServerAbilityInitializer

2026-04-01 12:20:30,893 INFO Load com.alibaba.nacos.core.ability.RemoteAbilityInitializer for ServerAbilityInitializer

2026-04-01 12:20:30,898 INFO The cluster resource is initialized

2026-04-01 12:20:31,052 INFO Get configCompatible config from env, ConfigCompatibleConfig{namespaceCompatibleMode=true}

2026-04-01 12:20:31,074 INFO [migrate] start migrate config namespace

2026-04-01 12:20:31,131 ERROR [migrate] config_info namespace migrate insert failed, retry times=0, error=PreparedStatementCallback; SQL [INSERT INTO config_info (data_id, group_id, content, md5, src_user, src_ip, app_name, tenant_id, c_desc, type, encrypted_data_key) select data_id, group_id, content, md5, ?, src_ip, app_name, 'public', c_desc, type, encrypted_data_key from config_info WHERE id IN (? ) ]; ERROR: null value in column "gmt_create" of relation "config_info" violates not-null constraint
  Detail: Failing row contains (7033, resource_config_IDENTITY__md.json, agentspec__enc.e6b395e59bbde592a8e8afa2e5b882e59cbae4b893e5aeb6_..., {"name":"IDENTITY.md","type":"config","content":"# 法国咨询..., c348a97c6c07d1fb569f8c0293d86694, null, null, nacos_namespace_migrate, null, , public, null, null, null, json, null, ).

2026-04-01 12:20:31,507 INFO Nacos Server is starting...

2026-04-01 12:20:32,178 ERROR [migrate] config_info namespace migrate insert failed, retry times=1, error=PreparedStatementCallback; SQL [INSERT INTO config_info (data_id, group_id, content, md5, src_user, src_ip, app_name, tenant_id, c_desc, type, encrypted_data_key) select data_id, group_id, content, md5, ?, src_ip, app_name, 'public', c_desc, type, encrypted_data_key from config_info WHERE id IN (? ) ]; ERROR: null value in column "gmt_create" of relation "config_info" violates not-null constraint
  Detail: Failing row contains (7034, resource_config_IDENTITY__md.json, agentspec__enc.e6b395e59bbde592a8e8afa2e5b882e59cbae4b893e5aeb6_..., {"name":"IDENTITY.md","type":"config","content":"# 法国咨询..., c348a97c6c07d1fb569f8c0293d86694, null, null, nacos_namespace_migrate, null, , public, null, null, null, json, null, ).

2026-04-01 12:20:32,507 INFO Nacos Server is starting...

2026-04-01 12:20:33,224 ERROR [migrate] config_info namespace migrate insert failed, retry times=2, error=PreparedStatementCallback; SQL [INSERT INTO config_info (data_id, group_id, content, md5, src_user, src_ip, app_name, tenant_id, c_desc, type, encrypted_data_key) select data_id, group_id, content, md5, ?, src_ip, app_name, 'public', c_desc, type, encrypted_data_key from config_info WHERE id IN (? ) ]; ERROR: null value in column "gmt_create" of relation "config_info" violates not-null constraint
  Detail: Failing row contains (7035, resource_config_IDENTITY__md.json, agentspec__enc.e6b395e59bbde592a8e8afa2e5b882e59cbae4b893e5aeb6_..., {"name":"IDENTITY.md","type":"config","content":"# 法国咨询..., c348a97c6c07d1fb569f8c0293d86694, null, null, nacos_namespace_migrate, null, , public, null, null, null, json, null, ).

2026-04-01 12:20:33,508 INFO Nacos Server is starting...

2026-04-01 12:20:34,270 ERROR [migrate] config_info namespace migrate insert failed, retry times=3, error=PreparedStatementCallback; SQL [INSERT INTO config_info (data_id, group_id, content, md5, src_user, src_ip, app_name, tenant_id, c_desc, type, encrypted_data_key) select data_id, group_id, content, md5, ?, src_ip, app_name, 'public', c_desc, type, encrypted_data_key from config_info WHERE id IN (? ) ]; ERROR: null value in column "gmt_create" of relation "config_info" violates not-null constraint
  Detail: Failing row contains (7036, resource_config_IDENTITY__md.json, agentspec__enc.e6b395e59bbde592a8e8afa2e5b882e59cbae4b893e5aeb6_..., {"name":"IDENTITY.md","type":"config","content":"# 法国咨询..., c348a97c6c07d1fb569f8c0293d86694, null, null, nacos_namespace_migrate, null, , public, null, null, null, json, null, ).

2026-04-01 12:20:34,509 INFO Nacos Server is starting...

2026-04-01 12:20:35,271 ERROR [migrate] config_info namespace migrate insert failed

2026-04-01 12:20:35,273 WARN Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'configChangeClusterSyncRequestHandler' defined in URL [jar:nested:/home/nacos/target/nacos-server.jar/!BOOT-INF/lib/nacos-config-3.2.0.jar!/com/alibaba/nacos/config/server/remote/ConfigChangeClusterSyncRequestHandler.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'configMigrateService': Invocation of init method failed

2026-04-01 12:20:35,291 WARN [WatchFileCenter] start close

2026-04-01 12:20:35,292 WARN [WatchFileCenter] start to shutdown this watcher which is watch : /home/nacos/conf

2026-04-01 12:20:35,293 WARN [WatchFileCenter] already closed

2026-04-01 12:20:35,293 INFO [NotifyCenter] Start destroying Publisher

2026-04-01 12:20:35,294 INFO [NotifyCenter] Completed destruction of Publisher

2026-04-01 12:20:35,294 ERROR Startup errors : 

org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'configChangeClusterSyncRequestHandler' defined in URL [jar:nested:/home/nacos/target/nacos-server.jar/!BOOT-INF/lib/nacos-config-3.2.0.jar!/com/alibaba/nacos/config/server/remote/ConfigChangeClusterSyncRequestHandler.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'configMigrateService': Invocation of init method failed
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:804)
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:240)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1395)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1232)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:569)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:529)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:339)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:373)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:337)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.instantiateSingleton(DefaultListableBeanFactory.java:1221)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingleton(DefaultListableBeanFactory.java:1187)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:1123)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:987)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:627)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:752)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:439)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:318)
	at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:149)
	at com.alibaba.nacos.bootstrap.NacosBootstrap.startCoreContext(NacosBootstrap.java:96)
	at com.alibaba.nacos.bootstrap.NacosBootstrap.startWithConsole(NacosBootstrap.java:84)
	at com.alibaba.nacos.bootstrap.NacosBootstrap.main(NacosBootstrap.java:54)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:102)
	at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:64)
	at org.springframework.boot.loader.launch.PropertiesLauncher.main(PropertiesLauncher.java:580)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'configMigrateService': Invocation of init method failed
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:222)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsBeforeInitialization(AbstractAutowireCapableBeanFactory.java:429)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1818)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:607)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:529)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:339)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:373)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:337)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:315)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:254)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1760)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1643)
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:913)
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:791)
	... 28 common frames omitted
Caused by: java.lang.Exception: [migrate] config_info namespace migrate insert failed
	at com.alibaba.nacos.config.server.service.ConfigMigrateService.doCheckNamespaceMigrate(ConfigMigrateService.java:533)
	at com.alibaba.nacos.config.server.service.ConfigMigrateService.migrate(ConfigMigrateService.java:157)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMethod.invoke(InitDestroyAnnotationBeanPostProcessor.java:457)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeInitMethods(InitDestroyAnnotationBeanPostProcessor.java:401)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:219)
	... 43 common frames omitted
2026-04-01 12:20:35,295 ERROR Nacos failed to start, please see /home/nacos/logs/nacos.log for more details.

2026-04-01 12:20:35,306 INFO 

Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.

2026-04-01 12:20:35,329 ERROR Application run failed

org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'configChangeClusterSyncRequestHandler' defined in URL [jar:nested:/home/nacos/target/nacos-server.jar/!BOOT-INF/lib/nacos-config-3.2.0.jar!/com/alibaba/nacos/config/server/remote/ConfigChangeClusterSyncRequestHandler.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'configMigrateService': Invocation of init method failed
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:804)
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:240)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1395)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1232)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:569)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:529)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:339)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:373)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:337)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.instantiateSingleton(DefaultListableBeanFactory.java:1221)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingleton(DefaultListableBeanFactory.java:1187)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:1123)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:987)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:627)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:752)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:439)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:318)
	at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:149)
	at com.alibaba.nacos.bootstrap.NacosBootstrap.startCoreContext(NacosBootstrap.java:96)
	at com.alibaba.nacos.bootstrap.NacosBootstrap.startWithConsole(NacosBootstrap.java:84)
	at com.alibaba.nacos.bootstrap.NacosBootstrap.main(NacosBootstrap.java:54)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:102)
	at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:64)
	at org.springframework.boot.loader.launch.PropertiesLauncher.main(PropertiesLauncher.java:580)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'configMigrateService': Invocation of init method failed
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:222)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsBeforeInitialization(AbstractAutowireCapableBeanFactory.java:429)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1818)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:607)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:529)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:339)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:373)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:337)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:315)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:254)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1760)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1643)
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:913)
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:791)
	... 28 common frames omitted
Caused by: java.lang.Exception: [migrate] config_info namespace migrate insert failed
	at com.alibaba.nacos.config.server.service.ConfigMigrateService.doCheckNamespaceMigrate(ConfigMigrateService.java:533)
	at com.alibaba.nacos.config.server.service.ConfigMigrateService.migrate(ConfigMigrateService.java:157)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMethod.invoke(InitDestroyAnnotationBeanPostProcessor.java:457)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeInitMethods(InitDestroyAnnotationBeanPostProcessor.java:401)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:219)
	... 43 common frames omitted
2026-04-01 12:20:35,332 INFO [HttpClientBeanHolder] Start destroying common HttpClient
```

--- a/plugin-default-impl/nacos-default-datasource-plugin/nacos-datasource-plugin-oracle/src/main/resources/META-INF/oracle-schema.sql
+++ b/plugin-default-impl/nacos-default-datasource-plugin/nacos-datasource-plugin-oracle/src/main/resources/META-INF/oracle-schema.sql
@@ -26,8 +26,8 @@ CREATE TABLE config_info (
     app_name VARCHAR2(128),
     content CLOB,
     md5 VARCHAR2(32) DEFAULT NULL,
-    gmt_create TIMESTAMP NOT NULL DEFAULT TO_TIMESTAMP('2010-05-05 00:00:00', 'YYYY-MM-DD HH24:MI:SS'),
-    gmt_modified TIMESTAMP NOT NULL DEFAULT TO_TIMESTAMP('2010-05-05 00:00:00', 'YYYY-MM-DD HH24:MI:SS'),
+    gmt_create TIMESTAMP NOT NULL DEFAULT SYSTIMESTAMP,
+    gmt_modified TIMESTAMP NOT NULL DEFAULT SYSTIMESTAMP,
     src_user VARCHAR2(128) DEFAULT NULL,
     src_ip VARCHAR2(50) DEFAULT NULL,
     c_desc VARCHAR2(256) DEFAULT NULL,
@@ -53,8 +53,8 @@ CREATE TABLE his_config_info (
     app_name VARCHAR2(128),
     content CLOB,
     md5 VARCHAR2(32) DEFAULT NULL,
-    gmt_create TIMESTAMP NOT NULL DEFAULT TO_TIMESTAMP('2010-05-05 00:00:00', 'YYYY-MM-DD HH24:MI:SS'),
-    gmt_modified TIMESTAMP NOT NULL DEFAULT TO_TIMESTAMP('2010-05-05 00:00:00', 'YYYY-MM-DD HH24:MI:SS'),
+    gmt_create TIMESTAMP NOT NULL DEFAULT SYSTIMESTAMP,
+    gmt_modified TIMESTAMP NOT NULL DEFAULT SYSTIMESTAMP,
     src_user VARCHAR2(128),
     src_ip VARCHAR2(50) DEFAULT NULL,
     publish_type VARCHAR2(50) DEFAULT 'formal',
@@ -82,8 +82,8 @@ CREATE TABLE config_info_gray (
     content CLOB,
     md5 VARCHAR2(32) DEFAULT NULL,
     encrypted_data_key VARCHAR2(256) DEFAULT NULL,
-    gmt_create TIMESTAMP NOT NULL DEFAULT TO_TIMESTAMP('2010-05-05 00:00:00', 'YYYY-MM-DD HH24:MI:SS'),
-    gmt_modified TIMESTAMP NOT NULL DEFAULT TO_TIMESTAMP('2010-05-05 00:00:00', 'YYYY-MM-DD HH24:MI:SS'),
+    gmt_create TIMESTAMP NOT NULL DEFAULT SYSTIMESTAMP,
+    gmt_modified TIMESTAMP NOT NULL DEFAULT SYSTIMESTAMP,
     CONSTRAINT configinfogray_id_key PRIMARY KEY (id),
     CONSTRAINT uk_configinfogray_datagrouptenantgrayname UNIQUE (data_id, group_id, tenant_id, gray_name)
 );
@@ -114,8 +114,8 @@ CREATE TABLE group_capacity (
     max_aggr_count NUMBER(10) DEFAULT 0,
     max_aggr_size NUMBER(10) DEFAULT 0,
     max_history_count NUMBER(10) DEFAULT 0,
-    gmt_create TIMESTAMP DEFAULT TO_TIMESTAMP('2010-05-05 00:00:00', 'YYYY-MM-DD HH24:MI:SS'),
-    gmt_modified TIMESTAMP DEFAULT TO_TIMESTAMP('2010-05-05 00:00:00', 'YYYY-MM-DD HH24:MI:SS'),
+    gmt_create TIMESTAMP DEFAULT SYSTIMESTAMP,
+    gmt_modified TIMESTAMP DEFAULT SYSTIMESTAMP,
     CONSTRAINT group_capacity_id_key PRIMARY KEY (id),
     CONSTRAINT uk_group_id UNIQUE (group_id)
 );
@@ -129,8 +129,8 @@ CREATE TABLE tenant_capacity (
     max_aggr_count NUMBER(10) DEFAULT 0,
     max_aggr_size NUMBER(10) DEFAULT 0,
     max_history_count NUMBER(10) DEFAULT 0,
-    gmt_create TIMESTAMP DEFAULT TO_TIMESTAMP('2010-05-05 00:00:00', 'YYYY-MM-DD HH24:MI:SS'),
-    gmt_modified TIMESTAMP DEFAULT TO_TIMESTAMP('2010-05-05 00:00:00', 'YYYY-MM-DD HH24:MI:SS'),
+    gmt_create TIMESTAMP DEFAULT SYSTIMESTAMP,
+    gmt_modified TIMESTAMP DEFAULT SYSTIMESTAMP,
     CONSTRAINT tenant_capacity_id_key PRIMARY KEY (id),
     CONSTRAINT uk_tenant_id UNIQUE (tenant_id)
 );
@@ -142,8 +142,8 @@ CREATE TABLE tenant_info (
     tenant_name VARCHAR2(128) DEFAULT '',
     tenant_desc VARCHAR2(256) DEFAULT NULL,
     create_source VARCHAR2(32) DEFAULT NULL,
-    gmt_create NUMBER(20) NOT NULL,
-    gmt_modified NUMBER(20) NOT NULL,
+    gmt_create TIMESTAMP NOT NULL DEFAULT SYSTIMESTAMP,
+    gmt_modified TIMESTAMP NOT NULL DEFAULT SYSTIMESTAMP,
     CONSTRAINT tenant_info_id_key PRIMARY KEY (id),
     CONSTRAINT uk_tenant_info_kptenantid UNIQUE (kp, tenant_id)
 );

--- a/plugin-default-impl/nacos-default-datasource-plugin/nacos-datasource-plugin-postgresql/src/main/resources/META-INF/pg-schema.sql
+++ b/plugin-default-impl/nacos-default-datasource-plugin/nacos-datasource-plugin-postgresql/src/main/resources/META-INF/pg-schema.sql
@@ -24,8 +24,8 @@ CREATE TABLE "config_info" (
   "group_id" varchar(255) ,
   "content" text  NOT NULL,
   "md5" varchar(32) ,
-  "gmt_create" timestamp(6) NOT NULL,
-  "gmt_modified" timestamp(6) NOT NULL,
+  "gmt_create" timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  "gmt_modified" timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
   "src_user" text ,
   "src_ip" varchar(20) ,
   "app_name" varchar(128) ,
@@ -143,8 +143,8 @@ CREATE TABLE "group_capacity" (
   "max_aggr_count" int4 NOT NULL,
   "max_aggr_size" int4 NOT NULL,
   "max_history_count" int4 NOT NULL,
-  "gmt_create" timestamp(6) NOT NULL,
-  "gmt_modified" timestamp(6) NOT NULL
+  "gmt_create" timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  "gmt_modified" timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP
 )
 ;
 COMMENT ON COLUMN "group_capacity"."id" IS '主键ID';
@@ -177,8 +177,8 @@ CREATE TABLE "his_config_info" (
   "app_name" varchar(128) ,
   "content" text  NOT NULL,
   "md5" varchar(32) ,
-  "gmt_create" timestamp(6) NOT NULL  DEFAULT '2010-05-05 00:00:00',
-  "gmt_modified" timestamp(6) NOT NULL,
+  "gmt_create" timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  "gmt_modified" timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
   "src_user" text ,
   "src_ip" varchar(20) ,
   "op_type" char(10) ,
@@ -245,8 +245,8 @@ CREATE TABLE "tenant_capacity" (
   "max_aggr_count" int4 NOT NULL,
   "max_aggr_size" int4 NOT NULL,
   "max_history_count" int4 NOT NULL,
-  "gmt_create" timestamp(6) NOT NULL,
-  "gmt_modified" timestamp(6) NOT NULL
+  "gmt_create" timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  "gmt_modified" timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP
 )
 ;
 COMMENT ON COLUMN "tenant_capacity"."id" IS '主键ID';
@@ -278,8 +278,8 @@ CREATE TABLE "tenant_info" (
   "tenant_name" varchar(128) ,
   "tenant_desc" varchar(256) ,
   "create_source" varchar(32) ,
-  "gmt_create" int8 NOT NULL,
-  "gmt_modified" int8 NOT NULL
+  "gmt_create" timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  "gmt_modified" timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP
 )
 ;
 COMMENT ON COLUMN "tenant_info"."id" IS 'id';

